centreon/www/front_src/src/Resources/Graph/Performance/AnomalyDetection/graph/AnomalyDetectionEstimatedEnvelopeThreshold.tsx

```diff
@@ -59,8 +59,9 @@ const AnomalyDetectionEstimatedEnvelopeThreshold = ({
     factors
   }: EnvelopeVariation): number => {
     return (
-      ((prop(metricUpper, item) as number) -
-        (prop(metricLower, item) as number)) *
+      (((prop(metricUpper, item) as number) -
+        (prop(metricLower, item) as number)) /
+        2) *
       (1 - factors.simulatedFactor / factors.currentFactor)
```

Comment:
I guess we should break it down by extracting into variables before returning the calculation